pkgs/games/anki/default.nix

```diff
@@ -1,199 +1,265 @@
-{ stdenv
-, buildPythonApplication
-, lib
-, python
-, fetchurl
+{ lib
```

Comment:
Please clean up unused parameters: `nodePackages`, `pkg-config`, `symlinkJoin` and `bash` (after removing the shebang usage).

Reply:
Done. Is there a linter that catches this which I should be running?

Reply:
> Is there a linter that catches this which I should be running?

I'm using nil LSP server which warns about it. https://github.com/oxalica/nil